_posts/en/posts/2024-07-03-disclose-unbounded-banlist.md

```diff
@@ -14,6 +14,7 @@ announcement: 1
 
 excerpt: >
   Bitcoin Core maintained an unlimited list of banned IP addresses and performed a quadratic operation on it. This could lead to an OOM crash and a CPU Dos.
+  A fix was released on August 1st, 2020 in Bitcoin Core 0.20.1.
```

Comment:
This excerpt is too long and gets cut off.

![Screenshot_20240708_115903](https://github.com/bitcoin-core/bitcoincore.org/assets/3782274/5b1c7cdf-f112-4383-9e25-fdf58159df76)

It looks like the advisories page has an excerpt character limit of 200, while the front page has a character limit of 160.

Several other excerpts are cutoff on the front page too.